app/Community/Components/UserProfileMeta.php

```diff
@@ -425,19 +426,30 @@ private function calculateAverageFinishedGames(array $userJoinedGamesAndAwards):
         return number_format($averageFinishedGames, 2, '.', '');
     }
 
-    private function calculateAveragePointsPerWeek(string $userMemberSince, int $points = 0): int
+    private function calculateAveragePointsPerWeek(User $user, bool $hardcore, int $points = 0): int
```

Comment:
```suggestion
    private function calculateAveragePointsPerWeek(User $user, bool $doesUserPreferHardcore, int $points = 0): int
```

nit: For a reader unfamiliar with the code, it may be unclear why the boolean arg would ever be set to true. I think renaming the arg could improve maintainability of this function.